btree/btree.c

```diff
@@ -1414,7 +1414,11 @@ static struct m0_rwlock list_lock;
 /**
  * Total space used by nodes in lru list.
  */
-static int64_t lru_space_used = 0;
+static int64_t lru_space_used;
+#ifndef __KERNEL__
+static bool lru_slow_purge;
```

Comment:
Who will turn ON and OFF this flag lru_slow_purge. I do not see it here.
In theory this flag should be turned ON the moment lru_space_used trips over the high_wm. This flag stays ON trickling lru nodes till the lru_space_used reaches below the target_wm so that no more trickling of lru nodes will happen until the node count in lru list again exceeds high_wm.

Reply:
This flag would be set only in the set() function along with the watermarks.
I think that we do not need to reset this flag after crossing below the target_wm as we will not be calling the purge function at all in that case and hence resetting this flag would be redundant. Please correct me if I am missing something.